app/resources/v1/pacbio/runs/pool_resource.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module V1
+  module Pacbio
+    module Runs
+      # PoolResource
+      class PoolResource < JSONAPI::Resource
+        model_name 'Pacbio::Pool'
+
+        has_many :libraries
+      end
+    end
+  end
+end
```

Comment:
Similar question to Library resource, what's the reasoning behind the separate run namespaced resource?